pest/src/iterators/pair.rs

```diff
@@ -248,6 +248,14 @@ impl<'i, R: RuleType> Pair<'i, R> {
     }
 }
 
+impl<'i, R: RuleType> Pairs<'i, R> {
+    /// Create a new `Pairs` iterator containing just the single `Pair`.
```

Comment:
Should probably contain a doc-test before merging in order to keep at least `pest`'s documentation fully complete.